src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -93,22 +93,22 @@ void GcsActorScheduler::ScheduleByGcs(std::shared_ptr<GcsActor> actor) {
                   .second);
 
     actor->SetAcquiredResources(ResourceMapToResourceRequest(
-        actor->GetCreationTaskSpecification().GetRequiredResources().GetResourceMap(),
-        false));
+        actor->GetLeaseSpecification().GetRequiredResources().GetResourceMap(), false));
     // Lease worker directly from the node.
     actor->SetGrantOrReject(true);
     LeaseWorkerFromNode(actor, node.value());
   };
 
   // Queue and schedule the actor locally (gcs).
   const auto &owner_node = gcs_node_manager_.GetAliveNode(actor->GetOwnerNodeID());
-  RayTask task(actor->GetCreationTaskSpecification(),
-               owner_node.has_value() ? actor->GetOwnerNodeID().Binary() : std::string());
-  cluster_task_manager_.QueueAndScheduleTask(std::move(task),
-                                             /*grant_or_reject*/ false,
-                                             /*is_selected_based_on_locality*/ false,
-                                             /*reply*/ reply.get(),
-                                             send_reply_callback);
+  RayLease lease(
+      actor->GetLeaseSpecification(),
+      owner_node.has_value() ? actor->GetOwnerNodeID().Binary() : std::string());
+  cluster_lease_manager_.QueueAndScheduleLease(std::move(lease),
+                                               /*grant_or_reject*/ false,
+                                               /*is_selected_based_on_locality*/ false,
+                                               /*reply*/ reply.get(),
```

Comment:
```suggestion
                                               /*grant_or_reject=*/false,
                                               /*is_selected_based_on_locality=*/false,
                                               reply.get(),
```
want to keep it consistent since the rest of the codebase uses equals in the comments

Reply:
Makes sense, done